content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -65,6 +67,12 @@ string HandleRollDice([FromServices]ILogger<Program> logger, string? player)
     return result.ToString(CultureInfo.InvariantCulture);
 }
 
+int RollDice()
+{
+    var rng = new Random();
+    return rng.Next(1, 6);
```

Comment:
Curious, is this the right way of using Random?

Reply:
Technically no, but that's more for when you've got a tight loop, since then it will be "less random" due to initializing each iteration. But I think per-request is fine for demonstrative purposes.

Reply:
While this is certainly fine for a demo why not use the same as in the auto-instrumentation docs?
https://github.com/open-telemetry/opentelemetry.io/blob/210e9c6c0d0279a7292e31bfe6fb54032ae44b87/content/en/docs/instrumentation/net/automatic/getting-started.md?plain=1#L54-L57

Reply:
mmm, yes, I had forgotten about the new .NET API. Let's go with that. @andrew-hillier can you make this change?